src/adapters/indexExchange.js

```diff
@@ -62,29 +340,26 @@ var IndexExchangeAdapter = function IndexExchangeAdapter() {
 						if (currentId === adSlotId) {
 							var bidObj = slotIdMap[adSlotId];
 							var adUnitCode = bidObj.placementCode;
-							var slotObj = getSlotObj(lookupObj, adSlotId);
+							var slotObj = getSlotObj(cygnus_index_args, adSlotId);
+
 							bid = bidfactory.createBid(1);
-							bid.cpm = (currentCPM / 100);
+							bid.cpm = currentCPM / 100;
 							bid.ad = adContents;
```

Comment:
Hi Matt,

Thanks for your help. That array will always have 1 element. So that can be changed to bid.ad = adContents[0];



From: mkendall <notifications@github.com<mailto:notifications@github.com>>
Reply-To: "prebid/Prebid.js" <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Thursday, December 3, 2015 at 3:17 PM
To: "prebid/Prebid.js" <Prebid.js@noreply.github.com<mailto:Prebid.js@noreply.github.com>>
Cc: casalej <jourdain.casale@casalemedia.com<mailto:jourdain.casale@casalemedia.com>>
Subject: Re: [Prebid.js] Index Exchange Adapter (#119)


In src/adapters/indexExchange.js<https://github.com/prebid/Prebid.js/pull/119#discussion_r46605435>:

>                                                       bid = bidfactory.createBid(1);
> -                                                     bid.cpm = (currentCPM / 100);
> +                                                     bid.cpm = currentCPM / 100;
>                                                       bid.ad = adContents;


@headertag<https://github.com/headertag>

I noticed adContents is actually an array with a single ad response. Is it always a single size array, and should we always take the first element from it?

-
Reply to this email directly or view it on GitHub<https://github.com/prebid/Prebid.js/pull/119/files#r46605435>.